packages/form/src/form-item.vue

```diff
@@ -5,7 +5,7 @@
     'is-required': isRequired || required
   }">
     <label :for="prop" class="el-form-item__label" v-bind:style="labelStyle" v-if="label">
```

Comment:
the slot won't be appended. and I think it is unnecessary to put label="" for this usage